### PR TITLE
Assume remote device when adding

### PR DESCRIPTION
**Pull request by nmattia**
This changes the "add device flow" (when started from an existing device) to optimistically assume the user wants to add a new browser (and hence entering "registration mode").

A button is added on this "add new browser" screen to opt-out and add a FIDO device instead.

### CODE REVIEW

File: src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts [hunk 1]
```diff
@@ -108,6 +117,8 @@ export const pollForTentativeDevicePage = renderPage(
   pollForTentativeDeviceTemplate
 );
 
+type PollReturn = DeviceData | "use fido" | "timeout" | "canceled";
```
[the remark this answers is not in the document]
Comment (nmattia) [reply src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts:120]: lol thought the same; I really don't care. I don't mind the space :)

File: src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts [hunk 2]
```diff
@@ -84,6 +86,13 @@ const pollForTentativeDeviceTemplate = ({
       <li>${copy.sign_in_using_device}</li>
     </ol>
 
+    <button
+      @click=${() => useFIDO()}
+      data-action="use-fido"
+      class="c-button c-button--primary l-stack"
+    >
+      ${copy.or_use_fido}
+    </button>
```
[the remark this answers is not in the document]
Comment (nmattia) [reply src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts:95]: yep makes sense!

File: src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts:120]: might be easier without the space for search and replace too. I can live with but if you don't mind removing the space, go for it

[the remark this answers is not in the document]
Comment (nmattia) [reply src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts:120]: ok changed, search and replace was actually a pain; very good point

**Pull request merged**